frameworks/keyed/yew/Cargo.toml

```diff
@@ -1,23 +1,23 @@
 [package]
-name = "js-framework-benchmark-yew"
-version = "1.0.0"
 authors = ["Isamu Mogi <isamu@leafytree.jp>"]
 edition = "2018"
+name = "js-framework-benchmark-yew"
+version = "1.0.0"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
-rand = { version = "0.7.3", features = ["wasm-bindgen", "small_rng"] }
+rand = {version = "0.7.3", features = ["wasm-bindgen", "small_rng"]}
 wasm-bindgen = "0.2.62"
 
 # UPDATE TO USE YOUR BRANCH
-yew = { git = "https://github.com/yewstack/yew", branch = "master" }
+yew = {path = "/home/janis/src/yew/packages/yew"}
```

Comment:
why was this path changed?

Reply:
Accidentally committed some dirty state. The benchmark script creates some as it runs.